Wikipedia/Code/MWKImage.h

```diff
@@ -46,6 +46,22 @@
 
 
 /**
+ * Width of the original image file. For example, if this image is a thumbnail, this is the width of the original image.
+ */
+@property (copy, nonatomic) NSNumber* originalFileWidth;
```

Comment:
I forgot to update the logic after making the change to separate properties. Here's that new change: 0a7a508a5100d798e46c06f67c79c3a5430f45b9